### PR TITLE
Add leadership model page

### PR DESCRIPTION
**Pull request by TimOsahenru**
Added a `meet-our-leaders/` URL to our nav links and used our logo as placeholder for those images not available, we can possibly create another issue to upload the respective pictures

### CODE REVIEW

File: _data/locales/en.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ en:
     - About Us
     - Events
     - Community
+    - Meet our leaders
```
Comment (kjaymiller) [inline _data/locales/en.yml]: I'd argue we don't need to add this page to the header. I do think we can add it to the footer and link to it from the about page.

Comment (kjaymiller) [inline _data/locales/en.yml]:
```suggestion
```

File: tests/test.py [hunk 2]
```diff
@@ -97,6 +99,7 @@ def test_switching_lang_sw_about(page: Page) -> None:
         ("Black Python Devs | About Us", "/about/"),
         ("Black Python Devs | Events", "/events/"),
         ("Black Python Devs | Community", "/community/"),
+        ("Black Python Devs | Meet our leaders", "/meet-our-leaders/"),
```
Comment (kjaymiller) [inline tests/test.py]:
```suggestion
```

**Comment by TimOsahenru** [comment]
@kjaymiller please help review this. Thanks

**Comment by kjaymiller** [comment]
@TimOsahenru I made some suggested changes

**Comment by TimOsahenru** [comment]
I don't see the suggested changes. Please point me to it

**Review by kjaymiller — CHANGES_REQUESTED** [review]

File: _config.yml [hunk 3]
```diff
@@ -30,6 +30,9 @@ navigation:
   - text: Community
     url: /community/
     fa: fa fa-users fa-fw
+  - text: Meet our leaders
```
Comment (kjaymiller) [inline _config.yml]:
```suggestion
```

File: meet-our-leaders.md [hunk 4]
```diff
@@ -0,0 +1,172 @@
+---
+layout: default
+lang: en
+title: Meet our leaders
```
Comment (kjaymiller) [inline meet-our-leaders.md]:
```suggestion
title: Leadership
```

File: _config.yml [hunk 5]
```diff
@@ -30,6 +30,9 @@ navigation:
...
+    url: /meet-our-leaders/
```
Comment (kjaymiller) [inline _config.yml]:
```suggestion
```

File: _config.yml [hunk 6]
```diff
@@ -30,6 +30,9 @@ navigation:
...
+    fa: fa fa-user-tie fa-fw
```
Comment (kjaymiller) [inline _config.yml]:
```suggestion
```

File: meet-our-leaders.md [hunk 7]
```diff
@@ -0,0 +1,172 @@
...
+<article class="leadership-photo-container">
+<img class="leadership-photo" alt="photo of Velda" src="https://github.com/VeldaKiara.png">
+<p><strong>Velda Kiara</strong><br/>
+Vice Executor</p>
+</article>
+<article class="leadership-photo-container">
+<img class="leadership-photo" alt="photo of Oleksis" src="https://github.com/oleksis.png">
+<p><strong>Oleksis Fraga</strong><br/>
+Technical Manager</p>
+</article>
```
Comment (kjaymiller) [inline meet-our-leaders.md]:
```suggestion
```

Comment (kjaymiller) [reply meet-our-leaders.md]: This is removed as per #463

File: meet-our-leaders.md [hunk 8]
```diff
@@ -0,0 +1,172 @@
...
+<p><strong>TBD</strong><br/>
+Content Team Coordinator</p>
+</article>
+
+<article class="leadership-photo-container">
+<img class="leadership-photo" alt="bpd logo" src="/assets/images/bpd_stacked.png">
+<p><strong>TBD</strong><br/>
+Code of Conduct Team Coordinator</p>
+</article>
+</div>
```
Comment (kjaymiller) [inline meet-our-leaders.md]: This should be a simplified list like our leadership council.

File: meet-our-leaders.md [hunk 9]
```diff
@@ -0,0 +1,172 @@
...
+<!-- end -->
+
+## Meetup Team
+
+The BPD Meetup Leaders are those responsible for the execution of our local meetups in select areas.
+
+<!-- Meetup team -->
+<div class="grid" style="display:flex; flex-wrap: wrap; justify-content:center;" markdown="1">
```
Comment (kjaymiller) [inline meet-our-leaders.md]: Simplify to a Markdown List

**Review by kjaymiller — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 6 above)
Comment (TimOsahenru) [reply _config.yml]: Can you please confirm the changes you want me to make in the `_config.yml` file. Do you want the `/meet-our-leaders/` completely removed or you want the name changed to *leadership* ?

Comment (kjaymiller) [reply _config.yml]: completely removed from the _config.yml as it's used for navigation.

File: tests/test.py [hunk 10]
```diff
@@ -6,9 +6,11 @@
 live_server_url = "http://127.0.0.1:4000"
 
 routes = [
+    ("blog"),
     ("about"),
-    ("community"),
     ("events"),
+    ("community"),
+    ("meet-our-leaders"),
```
Comment (kjaymiller) [inline tests/test.py]:
```suggestion
("leadership"),
```

**Review by kjaymiller — CHANGES_REQUESTED** [review]
There's still one small change in there which is an update to the test

File: _config.yml (hunk 6 above)
Comment (TimOsahenru) [reply _config.yml]: Noted. I'm curious as to how we plan to navigate to the leadership page if it isn't included in the navigation links.

File: tests/test.py (hunk 10 above)
Comment (TimOsahenru) [reply tests/test.py]: When I make this change, I encounter a failing test as we don't have a `leadership` route. Should I update the naming convention from `meet-our-leaders.md` to `leadership.md` instead? A 👍 for approval and I'll proceed.

**Review by kjaymiller — APPROVED** [review]

**Pull request merged**